packages/react-composites/tests/browser/call/hermetic/DtmfDialpad.test.ts

```diff
@@ -6,20 +6,34 @@ import { dataUiId, isTestProfileStableFlavor, pageClick, stableScreenshot, waitF
 import { buildUrlWithMockAdapter, defaultMockCallAdapterState, defaultMockRemoteParticipant, test } from './fixture';
 
 test.describe('Dtmf dialpad tests', async () => {
-  test('Dtmf dialpad should render correctly', async ({ page, serverUrl }) => {
+  test('Dtmf dialpad should render in 1:1 PSTN call', async ({ page, serverUrl }) => {
     test.skip(isTestProfileStableFlavor());
 
     const paul = defaultMockRemoteParticipant('Paul Bridges');
-    const vasily = defaultMockRemoteParticipant('Vasily Podkolzin');
 
-    const participants = [paul, vasily];
-    const initialState = defaultMockCallAdapterState(participants);
+    const participant = [paul];
+    const initialState = defaultMockCallAdapterState(participant);
 
+    //PSTN call has alternate caller id
+    initialState.alternateCallerId = '+1676568678999';
```

Comment:
Is this a "random" number not associated with a real phone number?
I am unsure whether or not we have numbers we can use for our testing.

Reply:
As long as we ensure that the number will not be a valid number ever, I am okay with this.